test/e2e/run-tests

```diff
@@ -54,6 +54,25 @@ function assert_value() {
   fi
 }
 
+function assert_value_within_range() {
+  # assert actual == expected within specified range
+  # intended for time comparisons
+  actual=$1
+  expected=$2
+  range=$3
+  difference=$(($actual - $expected))
+  if [[ $difference -lt 0 ]]; then
+    difference=$(expr $difference \* -1)
+  fi
+  echo "assert_value_within_range 4"
```

Comment:
what's 4 for?

Reply:
loll leftover from me tryna debug 'expr'; removing in next revision